ReleaseNotes.md

```diff
@@ -1,5 +1,64 @@
 # Release Notes
 
+## New in Release 2.13.0
+
+Post-training Quantization:
+
+- Breaking changes:
+  - ...
+- General:
+  - ...
+- Features:
+  - (OpenVINO) Added the ability to use the Scale Estimation algorithm to calculate quantization parameters in GPTQ for more accurate weight compression. The `gptq=True` and `scale_estimation=True` options can now be used together to enable calculation of quantization parameters using the Scale Estimation algorithm within GPTQ in `nncf.compress_weight`.
```

Comment:
PR: https://github.com/openvinotoolkit/nncf/pull/2948

Reply:
FYI, the `nncf.compress_weight` method doesn't exist. Only `nncf.compress_weights()`.